antora-playbook.yml

```diff
@@ -1,8 +1,10 @@
 antora:
   extensions:
+  - ./lib/antora/extensions/autoconfig-partial-build.js
```

Comment:
This extension detects when it's the appropriate time to do a partial build and reconfigures Atlas to accordingly.